aws_lambda_powertools/middleware_factory/__init__.py

```diff
@@ -1,4 +1,7 @@
-"""Utilities to enhance middlewares"""
+"""Utilities to enhance middlewares
```

Comment:
there's a typo here, there's no need to add an 's' to middleware

Reply:
Fixed